tests/scalability/sns/__init__.py

```diff
@@ -0,0 +1,21 @@
+import time
```

Comment:
Can you please try replacing these two files with a higher level Python threading API? I suggest using a simple concurrent.futures.map or threadpool context manager. It should make this code much simpler.

Reply:
Done